app/utils.ts

```diff
@@ -63,3 +64,29 @@ export function useUser(): User {
   }
   return maybeUser;
 }
+
+/**
+ * This async Fn takes a BullMQ job, and the queue name
+ * and returns children jobs that are of that queue name
+ * @param {string} jobName
+ * @param {Job} job
+ *
+ * @returns {Promise<Object>}
+ */
+
+export async function getChildrenValuesOfQueueName<CT>({
+  queueName,
+  job,
+}: {
+  queueName: string;
+  job: Job;
+}): Promise<{
+  [jobKey: string]: CT;
```

Comment:
Just curious about what the data type name CT is implying here. Is it related to certificates?